src/backend/core/api/viewsets.py

```diff
@@ -1048,7 +1100,20 @@ def search(self, request, *args, **kwargs):
             path_list |= db.Q(path__descendants=top_level_item)
 
         queryset = queryset.filter(path_list)
+
+        if indexer:
```

Comment:
It would be nice to be able to disable indexed search even if documents are indexed (to be able to start indexing document without actually use search on them).

=> Maybe open an issue if you don't want to do it in this PR.